credsweeper/utils/util.py

```diff
@@ -322,6 +339,16 @@ def is_pdf(data: bytes) -> bool:
                 return True
         return False
 
+    @staticmethod
+    def is_elf(data: Union[bytes, bytearray]) -> bool:
+        """According https://en.wikipedia.org/wiki/Executable_and_Linkable_Format - use only 5 bytes"""
```

Comment:
```suggestion
        """Accordingly to https://en.wikipedia.org/wiki/Executable_and_Linkable_Format use only 5 bytes"""
```

Reply:
https://github.com/babenek/CredSweeper/commit/c3526b2684030e09f8667e95bbb6a85e655e16f6 - will be applied in next pr